demos/text_detection_demo/src/text_recognition.cpp

```diff
@@ -25,6 +25,29 @@ namespace  {
         }
         *prob = 1.0f / static_cast<float>(sum);
     }
+
+    void softmax(const std::vector<float>::const_iterator& begin, const std::vector<float>::const_iterator& end, std::vector<float> &prob)
```

Comment:
Could you move `{` char to the same line as the function declaration?
(Everywhere in the demo this style is used.)